tests/test_interrupted_sync.py

```diff
@@ -0,0 +1,181 @@
+                            )
+
+                        if stream in {"pipelines", "workflows"}:
+                            repl_key = {
+                                "pipelines": "project_slug",
+                                "workflows": "pipeline_id",
+                            }
+
+                            for record in interrupted_records:
+                                rec_time = strptime_to_utc(record.get(replication_key))
+                                rec_repl_key = str(record[repl_key[stream]])
+                                interrupted_stream_state = bookmark_state[stream]
+                                if interrupted_stream_state["currently_syncing"] == rec_repl_key:
+                                    continue
+                                interrupted_bmk = interrupted_stream_state[rec_repl_key]
+                                self.assertGreaterEqual(rec_time, strptime_to_utc(interrupted_bmk))
+                        else:
+                            interrupted_bmk = strptime_to_utc(bookmark_state[stream][replication_key])
+                            for record in interrupted_records:
+                                rec_time = strptime_to_utc(record.get(replication_key))
+                                self.assertGreaterEqual(rec_time, strptime_to_utc(interrupted_bmk))
+
+                            # Record count for all streams of interrupted sync match expectations
+                            full_records_after_interrupted_bookmark = 0
+
+                            for record in full_records:
+                                rec_time = strptime_to_utc(record.get(replication_key))
+                                if rec_time >= strptime_to_utc(interrupted_bmk):
+                                    full_records_after_interrupted_bookmark += 1
+
+                            self.assertEqual(
+                                full_records_after_interrupted_bookmark,
+                                len(interrupted_records),
+                                msg=f"Expected {full_records_after_interrupted_bookmark} records in each sync",
+                            )
+
+                    elif stream in pending_streams:
+                        # First sync and second sync record count match
+                        self.assertGreaterEqual(
+                            second_sync_count,
+                            first_sync_count,
+                            msg="For pending sync streams - {}, second sync record count \
+                                                should be more than or equal to first sync".format(
+                                stream
+                            ),
+                        )
+                    else:
+                        raise Exception(f"Invalid state of stream {stream} in interrupted state")
+
+                elif expected_replication_method == self.FULL_TABLE:
+
```

Comment:
Additional FULL_TABLE verifications:
for rec in interrupted_records:
                        self.assertIn(rec, full_records, msg='full table record in interrupted sync not found in full sync')

                # Verify at least 1 record was replicated for each stream
                self.assertGreater(interrupted_record_count, 0)
                print(f"{stream} resumed sync records replicated: {interrupted_record_count}")